cdbot/cogs/cyber.py

```diff
@@ -375,15 +377,15 @@ def __init__(self, bot: Bot):
         exch_l_r = ctx.guild.get_role(ELITE_EXCH_LIST_ROLE_ID)
         exch_c_r = ctx.guild.get_role(ELITE_EXCH_CONF_ROLE_ID)
 
-
-        await ctx.send(f"""There are {len(game_r.members)} that have qualified for CyberStart Elite, with {len(exch_l_r.members)} qualifying for Exchange ({len(exch_c_r.members)} confirmed).
+        await ctx.send(f"""There are {len(elite_r.members)} that have qualified for CyberStart Elite.
+        {len(exch_l_r.members)} members have qualified for Exchange ({len(exch_c_r.members)} confirmed).
```

Comment:
I personally feel the mention of exchange is unnecessary - simply as it's a number that's unlikely to change, and one that can't really be influenced by others - although this is just an opinion

Reply:
Whilst I do see your point, this is more to reduce the number of messages I get asking how many people are in each role.

Reply:
Thank you Beano, very Pythonic!